dev/cc2520/cc2520.c

```diff
@@ -217,6 +217,7 @@ flushrx(void)
   uint8_t dummy;
 
   CC2520_READ_FIFO_BYTE(dummy);
+  (void)dummy;
```

Comment:
I suggest to either look for an alternative like defining `dummy`as `volatile` or at least add a comment why this line was added.